src/ProjectService/Controllers/TaskController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using LT.DigitalOffice.ProjectService.Business.Commands.Interfaces;
+using LT.DigitalOffice.ProjectService.Models.Dto.Requests;
+using LT.DigitalOffice.ProjectService.Models.Dto.RequestsModels;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.ProjectService.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class TaskController : ControllerBase
+    {
+        [HttpPost("createNewTask")]
```

Comment:
use just ''create'' because it's whatever clear what do you want to create by controller name, and url will be /task/create